src/AbstractUpdateAction.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Is this line automatically added to each file or is this something done manually?

Reply:
With the help of PHP-CS-Fixer `declare_strict_types` rule included in `@PHP80Migration:risky` rule set.

Reply:
Great, I was worried there would a be manual step involved.